Managers/WindowMan.cpp

```diff
@@ -318,19 +307,19 @@ namespace RTE {
 		}
 	}
 
-/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+	/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-	void WindowMan::ValidateResolution(int &resX, int &resY, int &resMultiplier) const {
+	void WindowMan::ValidateResolution(int &resX, int &resY, float &resMultiplier) const {
 		if (resX * resMultiplier > m_MaxResX || resY * resMultiplier > m_MaxResY || resMultiplier < 1 || resMultiplier > 8) {
```

Comment:
Let's delete the resMultipler < 1 or > 8 checks now, I guess, they seem unnecessary